tools/dep_updaters/update-inspector-protocol.sh

```diff
@@ -37,7 +37,7 @@ python "$BASE_DIR/tools/inspector_protocol/roll.py" \
   --node_src_downstream "$BASE_DIR" \
   --force
 
-NEW_VERSION=$(grep "Revision:" "$DEPS_DIR/inspector_protocol/README.node" | sed -n "s/^Revision: \(\\w*\)/\1/p")
+NEW_VERSION=$(grep "Revision:" "$DEPS_DIR/inspector_protocol/README.node" | sed -n "s/^Revision: \(\\w*\)/\1/p" | head -c 36)
```

Comment:
nit: we don't need to spawn the additional `head` process
```suggestion
NEW_VERSION=$(grep "Revision:" "$DEPS_DIR/inspector_protocol/README.node" | sed -n \([[:xdigit:]]\{36\}\).*$/\1/p" )
```